### PR TITLE
Fallout New Vegas

### PR DESCRIPTION
**Pull request by alsoandanswer**
- rscadd: "Slot machines now accept credits! Use the multitool to switch between coin mode and credit mode."
  - soundadd: "Slot machines also have sound! Be sure to toggle Arcade SFX to hear it. Also, makes it 87% more addictive."
  (Taken from Freesound. Cheesily recognisable stock sound effects. But damn, it is satisfying to hear.)
  - imageadd: "Adds new sprites for slot machines."

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by SleepyWolfe** [comment]
will this add a multitool to the bar as well?

**Comment by Caelphon** [comment]
@SleepyWolfe I believe if it isn't it can be located within the Auxiliary Tool Storage on the surface opposite of the elevators - if I am mistaken, could always bug a engineer.

**Comment by alsoandanswer** [comment]
!wip need to fix roundstart empty slots

**Comment by alsoandanswer** [comment]
!review

**Review by Casper3667 — APPROVED** [review]

File: code/game/machinery/computer/slotmachine.dm [hunk 1]
```diff
@@ -2,74 +2,82 @@
...
-
-	else if(stat & BROKEN)
-		icon_state = "slotsb"
-
-	else if(working)
-		icon_state = "slots2"
-
+	if(stat & BROKEN)
+		icon_state = "slots_broken"
+		if(stat & NOPOWER)
```
Comment (Arrow768) [inline code/game/machinery/computer/slotmachine.dm:70]: This condition (`stat & NOPOWER`) unless the previous condition is also triggered (`stat & BROKEN`)

Shouldn't be first condition be something like `(stat & BROKEN || stat & NOPOWER)` ?

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**